catavatar/catavatar.php

```diff
@@ -168,64 +178,65 @@ function catavatar_module(){}
  *
  * @TODO: support sizes
  */
-function catavatar_content($a) {
-	if ($a->argc < 2 || $a->argc > 3)
+function catavatar_content($a)
+{
+	if ($a->argc < 2 || $a->argc > 3) {
 		throw new NotFoundException(); // this should be catched on index and show default "not found" page.
+	}
 
 	$uid = intval($a->argv[1]);
-	
+
 	$size = 0;
 	if ($a->argc == 3) {
 		$size = intval($a->argv[2]);
 	}
-	
+
 	$user = dba::selectFirst('user', ['email'],
 		[
 			'uid' => $uid,
 			'blocked' => 0,
 			'account_expired' => 0,
 			'account_removed' => 0,
 		]
-	);	
-	
-	if ($user === False)
+	);
+
+	if ($user === false) {
 		throw new NotFoundException();
-	
+	}
+
 	$seed = PConfig::get(local_user(), "catavatar", "seed", md5(trim(strtolower($user['email']))));
-	//echo "<pre>"; var_dump($hash); killme();
 
-	
 	// from cat-avatar-generator.php
-
-	$imageurl = $seed."-".$size;
-	$imageurl = preg_replace('/[^A-Za-z0-9\._-]/', '', $imageurl); 
-	$imageurl = substr($imageurl,0,35).'';
+	$imageurl = $seed . "-" . $size;
+	$imageurl = preg_replace('/[^A-Za-z0-9\._-]/', '', $imageurl);
+	$imageurl = substr($imageurl,0,35) . '';
 	$cachefile = get_cachefile($imageurl);
 	$cachetime = 604800; # 1 week (1 day = 86400)
 
 	// Serve from the cache if it is younger than $cachetime
-	if ($cachefile != "" && file_exists($cachefile) && time() - $cachetime < filemtime($cachefile)) {
+	if ($cachefile != "" && file_exists($cachefile) && (time() - $cachetime) < filemtime($cachefile)) {
 		header('Pragma: public');
 		header('Cache-Control: max-age=86400');
 		header('Expires: '. gmdate('D, d M Y H:i:s \G\M\T', time() + 86400));
 		header('Content-Type: image/jpg');
 		readfile($cachefile);
-		exit;
+		killme();
```

Comment:
Obsolete: `killme()` is deprecated and will be removed in a future version, `exit()` was fine.